docs/user-guide/src/bmo/bmo.md

```diff
@@ -0,0 +1,98 @@
+<h1>Bare Metal Operator</h1>
+
+The Bare Metal Operator (BMO) is a custom Kubernetes controller that deploys baremetal hosts, represented by BMH, as Kubernetes nodes. To this end Ironic is used.
+
+BMO can also work with the Cluster API Provider Metal3 (CAPM3) controller. With the involvment of CAPM3 and Ironic, a simplified information flow path is shown below:
+
+The BareMetalHost represents a bare metal host (server). The BareMetalHost contains information about the server as shown below.
+For brevity, some part of the output are omitted, but we can classify the fields into the following broad categoreis.
+
+(1) known server properies: Fields such as `bootMACAddress` properies of the server and are known in advance.
+(2) unknown server properties: Fields such as `CPU` and `disk` are properties of the server and are discovered by Ironic.
+(3) user supplied: Fields such as `image` are supplied by user to dictate boot image for the server.
+(4) dynamic fields: Fields such as IP could be dynamically assigned to the server at run time by dhcp server.
```

Comment:
I suggest doing this as a markdown list. Otherwise it will just end up as a single paragraph that can be a bit hard to read.
```suggestion
1. known server properies: Fields such as `bootMACAddress` properies of the server and are known in advance.
2. unknown server properties: Fields such as `CPU` and `disk` are properties of the server and are discovered by Ironic.
3. user supplied: Fields such as `image` are supplied by user to dictate boot image for the server.
4. dynamic fields: Fields such as IP could be dynamically assigned to the server at run time by dhcp server.
```

Reply:
done